docs/advanced/profiling.rst

```diff
@@ -142,7 +142,7 @@ to report sections of the code for timing. To enable the section reporting, use
 CMake option during configuration.
 
 
-.. _AMD OmniTrace: https://amdresearch.github.io/omnitrace/index.html
+.. _AMD OmniTrace: https://rocm.docs.amd.com/projects/rocprofiler-systems/en/latest/
```

Comment:
Yep, AMD renamed all their tooling related to the profiling systems, so I actually don't know how accurate any of this AMD stuff is as of right now. When I spoke to some people at SIAM CSE last week, it sounded like they actually collapsed the omnitrace API here into rocTX for the rocprofiler-system, so it might be this separate part isn't needed anymore.